HW03/aleksei_zhuchkov/spec/hw_03_t_01_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../aleksei_zhuchkov_hw_03_t_01'
+
+RSpec.describe 'Homework3' do
+  let(:obj_hw3) { Homework3.new }
+
+  describe 'testing basics things' do
```

Comment:
it's better to use here method names

Reply:
please, use contexts and normal logs instead of 1 word

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/3483c4bde37292ce6fb967100b433f4ba9e659f6